receiver.go

```diff
@@ -556,6 +582,9 @@ func (r *Receiver) mux(hooks receiverTestHooks) {
 				return
 			}
 
+		case fr := <-txDisposition:
+			_ = r.l.txFrame(fr)
```

Comment:
If we ignore this error does that mean the user will never get any notification that the disposition wasn't sent?

Reply:
More onion and highlights a deeper problem I've been thinking about while working on this change.

In this example, if there's no error, it only means that we sent the frame to the session; this doesn't necessarily mean that the frame was written to the socket (we have this problem elsewhere).

You might be tempted to add `<-conn.done` to the `select` however that won't solve the problem.

I think some more work is in order.

Reply:
I have a fix for this, the "reliable writes" feature I've been talking about.  It would be made post this PR.